src/openrct2/peep/Peep.cpp

```diff
@@ -1860,18 +1860,18 @@ void Peep::FormatActionTo(void* argsV) const
             ft.Add<rct_string_id>(STR_SITTING);
             break;
         case PEEP_STATE_WATCHING:
-            if (current_ride != RIDE_ID_NULL)
+            if (CurrentRide != RIDE_ID_NULL)
             {
-                auto ride = get_ride(current_ride);
+                auto ride = get_ride(CurrentRide);
                 if (ride != nullptr)
                 {
-                    ft.Add<rct_string_id>((current_seat & 0x1) ? STR_WATCHING_CONSTRUCTION_OF : STR_WATCHING_RIDE);
+                    ft.Add<rct_string_id>((CurrentSeat & 0x1) ? STR_WATCHING_CONSTRUCTION_OF : STR_WATCHING_RIDE);
```

Comment:
Should be `standing_flags` (which is in a union with `CurrentSeat`)

Reply:
Nice catch!